packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -297,18 +296,6 @@ describeCompat("Named root data stores", "FullCompat", (getTestObjectProvider) =
 			assert.ok(await getAliasedDataStoreEntryPoint(dataObject1, alias));
 		});
 
-		it("Sending a bad alias message returns error", async () => {
-			try {
-				(runtimeOf(dataObject1) as ContainerRuntime).submitDataStoreAliasOp(
-					{ id: alias },
-					undefined,
-				);
-				assert.fail("Expected exception from sending invalid alias");
-			} catch (err) {
-				assert.equal((err as Error).message, "malformedDataStoreAliasMessage");
-			}
-		});
-
```

Comment:
Obviously, I'm curious about how aliasing will work if we try to do it at lower layers

Reply:
At the moment, I do not want to expose it, but I think it will be local to a DataStores instance, so it will work exactly the same way as it does (but one layer down).
This test though is trying to do something weird - it throws garbage op (using internal API) and hopes for non-garbage result. Internal API is gone, and with that - check that existed on ContainerRuntime before sending op. It would still fail (if you were to send same garbage), but on receiving side.
I do not see much value in recreating this test.